docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>Do we use column 0 later? If not, then it might be better to just read the csv for that column only. So, we can do:</p><pre class="ql-syntax" spellcheck="false">T = pd.read_csv(..., usecols=["1"]).to_numpy(np.float64)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I modified the dataset itself, it was originally exstracted dataset.</p>